panic: runtime error: index out of range (within IntelliJ)

1. 
Delve Debugger
Version: 0.11.0-alpha
Build: c7f11149d7146427b96be5375aff10d5bc899f5a

2. 
go version go1.7 darwin/amd64

3. Mac OSX (see above)

4. Debugging within IntelliJ.  Set a few breakpoints.  Things are working until I hit a specific breakpoint within a method which looks like this: 	for _, blah := range jsonData {

5. Nothing.

6. Debugger bombs: Process finished with exit code 2: 

panic: runtime error: index out of range

goroutine 36 [running]:
github.com/derekparker/delve/proc.(*Variable).loadInterface(0xc820099b20, 0x1, 0x44ca001, 0x1, 0x1, 0x40, 0x40, 0xffffffffffffffff)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:1518 +0x1639
github.com/derekparker/delve/proc.(*Variable).loadValueInternal(0xc820099b20, 0x1, 0x1, 0x1, 0x40, 0x40, 0xffffffffffffffff)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:788 +0xc5f
github.com/derekparker/delve/proc.(*Variable).loadArrayValues(0xc820099880, 0x0, 0xc82107f101, 0x1, 0x40, 0x40, 0xffffffffffffffff)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:964 +0x2f1
github.com/derekparker/delve/proc.(*Variable).loadValueInternal(0xc820099880, 0x0, 0xc820712301, 0x1, 0x40, 0x40, 0xffffffffffffffff)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:766 +0x98a
github.com/derekparker/delve/proc.(*Variable).loadValue(0xc820099880, 0xc821653001, 0x1, 0x40, 0x40, 0xffffffffffffffff)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:726 +0x5b
github.com/derekparker/delve/proc.(*EvalScope).extractVariableFromEntry(0xc820ab8360, 0xc821653050, 0x1, 0x1, 0x40, 0x40, 0xffffffffffffffff, 0xc8200995e0, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:505 +0xdb
github.com/derekparker/delve/proc.(*EvalScope).variablesByTag(0xc820ab8360, 0x34, 0xc8225c1801, 0x1, 0x40, 0x40, 0xffffffffffffffff, 0x0, 0x0, 0x0, ...)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:1582 +0x1b8
github.com/derekparker/delve/proc.(*EvalScope).LocalVariables(0xc820ab8360, 0xffffffffffffff01, 0x1, 0x40, 0x40, 0xffffffffffffffff, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/variables.go:536 +0x93
github.com/derekparker/delve/service/debugger.(*Debugger).LocalVariables(0xc8200116c0, 0xffffffffffffffff, 0x0, 0x1, 0x1, 0x40, 0x40, 0xffffffffffffffff, 0x0, 0x0, ...)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/service/debugger/debugger.go:630 +0x17b
github.com/derekparker/delve/service/rpc1.(*RPCServer).ListLocalVars(0xc8224d5aa0, 0xffffffffffffffff, 0x0, 0xc820ab8340, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/service/rpc1/server.go:211 +0x9b
reflect.Value.call(0x456f0e0, 0x46601f0, 0x13, 0x467ab48, 0x4, 0xc822699f18, 0x3, 0x3, 0x0, 0x0, ...)
	/usr/local/go/src/reflect/value.go:432 +0x120a
reflect.Value.Call(0x456f0e0, 0x46601f0, 0x13, 0xc822699f18, 0x3, 0x3, 0x0, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:300 +0xb1
github.com/derekparker/delve/service/rpccommon.(*ServerImpl).serveJSONCodec(0xc820019130, 0x709d0a8, 0xc820030108)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/service/rpccommon/server.go:287 +0xb35
created by github.com/derekparker/delve/service/rpccommon.(*ServerImpl).Run.func1
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/service/rpccommon/server.go:145 +0x185

goroutine 1 [chan receive]:
github.com/derekparker/delve/cmd/dlv/cmds.execute(0x0, 0xc820017b80, 0x1, 0x4, 0xc8200300c8, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/cmd/dlv/cmds/commands.go:428 +0xad5
github.com/derekparker/delve/cmd/dlv/cmds.New.func4(0xc820090e00, 0xc820017b80, 0x1, 0x4)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/cmd/dlv/cmds/commands.go:154 +0x53
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).execute(0xc820090e00, 0xc820017a80, 0x4, 0x4, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:569 +0x869
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc820090600, 0xc820090e00, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:656 +0x56b
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).Execute(0xc820090600, 0x0, 0x0)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:615 +0x2d
main.main()
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/cmd/dlv/main.go:13 +0x4a

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 7 [chan receive, locked to thread]:
github.com/derekparker/delve/proc.(*Process).handlePtraceFuncs(0xc82008e500)
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/proc.go:774 +0xc3
created by github.com/derekparker/delve/proc.New
	/Users/builduser/buildAgent/work/b97f8195591f35ac/gopath/src/github.com/derekparker/delve/proc/proc.go:82 +0x1d0

goroutine 35 [select, locked to thread]:
runtime.gopark(0x478c6a8, 0xc82072f728, 0x46836a8, 0x6, 0x18, 0x2)
	/usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc82072f728, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc82072f728)
	/usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:227 +0x323
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

Process finished with exit code 2

--- a/proc/variables.go
+++ b/proc/variables.go
@@ -1513,6 +1513,7 @@ func (v *Variable) loadInterface(recurseLevel int, loadData bool, cfg LoadConfig
 		typdata, err := v.dbp.CurrentThread.readMemory(res, 3+v.dbp.arch.PtrSize())
 		if err != nil {
 			v.Unreadable = fmt.Errorf("could not read concrete type (data: %#v): %v", data.Addr, err)
+			return
 		}
 
 		nl := int(typdata[1]<<8 | typdata[2])

